src/index.js

```diff
@@ -0,0 +1,55 @@
+import './index.scss';
+import leftMenuDiv from './views/javascripts/left-menu';
+import { mainDiv, TaskList } from './views/javascripts/main';
+import DefaultTask from './views/javascripts/_default-values';
+
+const listName = 'default';
+// eslint-disable-next-line
+let bodyStructure = null;
+
+if (!localStorage.getItem('lastConnection')) {
+  localStorage.setItem('lastConnection', 'default');
+}
+
+/** Crete the body structure */
+const BodyStructure = () => {
+  const arrElements = [];
+
+  const header = document.createElement('header');
+  arrElements.push(header);
+  const leftMenu = document.createElement('div');
+  leftMenu.className = 'left-menu';
+  arrElements.push(leftMenu);
+  const main = document.createElement('main');
+  arrElements.push(main);
+  const footer = document.createElement('footer');
+  arrElements.push(footer);
+
+  arrElements.forEach((element) => {
+    document.body.appendChild(element);
+  });
+
+  return {
+    header, leftMenu, main, footer,
+  };
+};
+
+/*
+function local() {
+  Object.keys(localStorage).forEach(i => {
+    localStorage.removeItem(i);
+  });
+}  */
```

Comment:
Please remove commented out code.
 It’s better to delete commented code. If you need to review a previous version of your code, use git history
Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from pathwright